### DISCUSSION TITLE
PackIcon failed to show

### DISCUSSION THREAD

**Discussion by hxhgxy** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2843] [Q&A] [answered]
PackIcon in Icon Button is shown as expected in design mode, but it failed to show in running mode. what i can do?
Thanks,
Tiger

**Comment by Keboo** [comment]
I am not able to reproduce this issue, could you please provide a code sample, which version of Visual Studio you are using, and which version of this library you are referencing.

**Comment by hxhgxy** [comment, marked as answer]
there are many similar bugs related to VS, I fixed the problem by adding reference in app.xaml. Sometimes, the generic.xaml works bad.
